### DISCUSSION TITLE
External validation farmer/public key/wallet is alive on chiaexplorer.com

### DISCUSSION THREAD

**Discussion by Isanderthul** [Chia-Network/chia-blockchain#5391] [Q&A]
How can I see my public key on chiaexplorer.com?
How can I verify externally if my farmer is actually communicating with the community?
Can I see a zero value wallet on chiaexplorer.com?

All these are how to validate everything is working from an external perspective, not using the `~/.chia/mainnet/log/debug.log`

**Comment by lvcivs** [comment]
1. You can search for a public key here: https://www.chiaexplorer.com/blockchain/search
2. You can look up your wallet address at the mentioned link. Note that your wallet will show up only once it has received something. So if you have not farmed any blocks yet, you can use the chia faucet to receive a tiny amount of chia and then it will show up.
3. No, if your wallet has not received any chia, it will not show up.

**Reply by Isanderthul** [reply]
Thank you @lvcivs I have used the https://faucet.chia.net/, and I got 0.0000000001 XCH into my wallet!

**Reply by firmicus** [reply]
Thank you @lvcivs
